docs/submission/submitting-molecular-data.md

```diff
@@ -57,78 +57,161 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ## How to Upload Molecular Data
 
-### 1. Format the payload
+### Understanding Song payload fields
 
-/////// NEED THE TAB DISPLAY HERE TO SHOW DIFFEERENT STARTING PAYLOADS {SEQUENCING EXPEIRMETN FOR ONE}
+Song accepts data in JSON format, which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare SONG metadata payloads conforming to most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+
+The data fields can be broken down into four main sections: `experiment`, `sample`, `read groups` and `files`.
+
+> Note: \*\* denotes a required field. These must be provided as part of the metadata payload or it will immediately fail validation upon submission.
+
+#### **Experiment:** The experiment section contains details relevant the experimental requirements imposed during sequencing.
+
+- `**studyId`: Corresponds to your ARGO `Program ID`. This is the same Program ID that you see when you log in to the ARGO Data Platform.
+- `**submitter_sequencing_experiment_id`:The unique identifier of the sequencing experiment.
+- `**platform`: The sequencing platform type that was used to generate the submitted data files.
+- `platform_model`: The exact model number of the sequencing machine used.
+- `sequencing_center`: The sequencing center the analysis was performed at.
+- `**experimental_strategy`: Descriptor of the read domain experiment method; Sometimes also referred to as library strategy.
+- `sequencing_date`: Date sequencing was performed.
+- `read_group_count`: Number of read groups submitted as part of the raw molecular file.
+
+**Sample:** The sample section contains details of the clinical data and key sample descriptors related to the submitted files. In order to submit a payload, this data must already be [registered](../submission/registering-samples) to the ARGO Data Platform. For allowed values of all fields, please see the Sample Registration file of the [Data Dictionary](/dictionary)

```

Comment:
change  this data must already be registered to

this `sample` must already be registered to

Reply:
allowed is more accurate.

Reply:
it's just such a bossy word.
I like permitted....or required is what is used in the dictionary. that's why I suggested it